### DISCUSSION TITLE
Error after installing

### DISCUSSION THREAD

**Discussion by logan1o1** [air-verse/air#610] [Q&A] [answered]
I installed air using:-  go install github.com/air-verse/air@latest , and my golang version is 1.22.3. but when trying to run it on the terminal it is giving this error: 
air : The term 'air' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, 
verify that the path is correct and try again.
what to do

**Comment by xiantang** [comment]
are u using windows?

**Comment by logan1o1** [comment, marked as answer]
Problem was solved later
